src/lolex-src.js

```diff
@@ -710,6 +710,15 @@ function createClock(start, loopLimit) {
 
     if (performancePresent) {
         clock.performance = Object.create(global.performance);
```

Comment:
I originally wanna write code like this,
```javascript
clock.performance = Object.create(global.Performance.prototype);
```

In Phantom.js environment, prototype's properties that be assigned by developer didn't be copied for unknown reason.